pkg/coscheduling/core/core.go

```diff
@@ -91,17 +85,15 @@ type PodGroupManager struct {
 }
 
 // NewPodGroupManager creates a new operation object.
-func NewPodGroupManager(pgClient pgclientset.Interface, snapshotSharedLister framework.SharedLister, scheduleTimeout, deniedPGExpirationTime *time.Duration,
+func NewPodGroupManager(pgClient pgclientset.Interface, snapshotSharedLister framework.SharedLister, scheduleTimeout *time.Duration,
 	pgInformer pginformer.PodGroupInformer, podInformer informerv1.PodInformer) *PodGroupManager {
 	pgMgr := &PodGroupManager{
-		pgClient:                   pgClient,
-		snapshotSharedLister:       snapshotSharedLister,
-		scheduleTimeout:            scheduleTimeout,
-		lastDeniedPGExpirationTime: deniedPGExpirationTime,
-		pgLister:                   pgInformer.Lister(),
-		podLister:                  podInformer.Lister(),
-		lastDeniedPG:               gochache.New(3*time.Second, 3*time.Second),
-		permittedPG:                gochache.New(3*time.Second, 3*time.Second),
+		pgClient:             pgClient,
+		snapshotSharedLister: snapshotSharedLister,
+		scheduleTimeout:      scheduleTimeout,
+		pgLister:             pgInformer.Lister(),
+		podLister:            podInformer.Lister(),
+		permittedPG:          gochache.New(3*time.Second, 3*time.Second),
```

Comment:
Can we get a data point on how the integration test behave w/ vs. w/o permittedPG?

Reply:
Sure here is data point for with and without permittedPG

with
```
time make integration-test
hack/install-envtest.sh
hack/integration-test.sh
+++ [0721 18:32:17] Configuring envtest
+++ [0721 18:32:17] Running integration test cases
ok  	sigs.k8s.io/scheduler-plugins/test/integration	250.299s
```

without
```
time make integration-test
hack/install-envtest.sh
hack/integration-test.sh
+++ [0721 18:42:05] Configuring envtest
+++ [0721 18:42:05] Running integration test cases
ok  	sigs.k8s.io/scheduler-plugins/test/integration	253.303s
```

I think the `permittedPG` is useful to avoid having to check [cluster resource](https://github.com/kubernetes-sigs/scheduler-plugins/blob/master/pkg/coscheduling/core/core.go#L188) for every pod in PreFilter. Basically with this, it only checks once for a pod in the PodGroup and the other pods don't need to check again and can pass through PreFilter